packages/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -806,105 +785,8 @@ test("Case-sensitive tag has capitalization preserved", function() {
   compilesTo('<svg><linearGradient id="gradient"></linearGradient></svg>');
 });
 
-// QUnit.skip("svg can live with hydration", function() {
-//   let template = compile('<svg></svg>{{name}}');
-
-//   let fragment = render(template, { name: 'Milly' }, env, { contextualElement: document.body }).fragment;
-
-//   equal(
-//     fragment.childNodes[0].namespaceURI, svgNamespace,
-//     "svg namespace inside a block is present" );
-// });
-
-// QUnit.skip("top-level unsafe morph uses the correct namespace", function() {
```

Comment:
Moved to `updating-test.ts` as INUR, renamed `<svg> tag and unsafe expression as sibling`